fixtures/server-config.rethink.json

```diff
@@ -0,0 +1,26 @@
+{
+	"server": {
+		"http_addr": ":4443",
+		"tls_key_file": "./notary-server.key",
+		"tls_cert_file": "./notary-server.crt"
+	},
+	"trust_service": {
+	  "type": "remote",
+	  "hostname": "notarysigner",
+	  "port": "7899",
+	  "tls_ca_file": "./root-ca.crt",
+	  "key_algorithm": "ecdsa",
+	  "tls_client_cert": "./notary-server.crt",
+	  "tls_client_key": "./notary-server.key"
+	},
+	"logging": {
+		"level": "debug"
+	},
+	"storage": {
+		"backend": "rethinkdb",
+		"auth_key": "devauthkey",
+		"db_url": "rdb-proxy.rdb",
+		"database": "notaryserver",
+		"tls_ca_file": "/tls/ca.pem"
```

Comment:
Non-blocking - would it make sense to make this a relative link instead, if we allow relative links for this?  Since a reader would have to keep in mind that this can only be used in the context of the container as specified by one of our rethink compose files.